SG14/plf_colony.h

```diff
@@ -186,7 +186,7 @@
 #include <limits>  // std::numeric_limits
 #include <memory>	// std::allocator
 #include <iterator> // std::bidirectional_iterator_tag
-
+#include <cstddef>

```

Comment:
Maybe add a comment `// offsetof` like for other includes?

Reply:
Good point, thx